environments/production/inventory.ini

```diff
@@ -81,6 +87,7 @@ hqdb0
 hqdb1
 hqdb2
 hqdb3
+hqdb5
```

Comment:
what about hqdb4?

Reply:
It was an oversight. Didn't care much since this was going to be hqdb2 after the migration

Reply:
what do you mean it's going to be hqdb2? long term plan was just splitting the current database I thought?

Reply:
From the doc, I gathered that we wanted to increase the resources on hqdb2 by migrating all of it to a flavour where we can add resources. If that's not the case, then we should probably just destroy this VM and restart, because this VM has double the resources of hqdb2 which it probably doesn't need if we are splitting hqdb2 and since this is not resizable as well.